Ten Elephants/Ten Elephants/Controllers/TrendPageController.swift

```diff
@@ -267,5 +279,19 @@ extension TrendPageController: UICollectionViewDataSource {
             } else {
                 return UICollectionReusableView()
             }
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        guard let section = Section(rawValue: indexPath.section) else { fatalError("No section provided") }
```

Comment:
fatal слишком жестко для такого сценария
```suggestion
        guard let section = Section(rawValue: indexPath.section) else { assertionFailure("No section provided") }
```